src/components/cylc/tree/Tree.vue

```diff
@@ -6,15 +6,14 @@
         :key="workflow.id"
         :node="workflow"
         :hoverable="hoverable"
-        :min-depth="minDepth"
```

Comment:
Not using `min-depth` any longer. We are now simply giving the component an array with the cycle points. Each cycle point will contain families or tasks, each family contains tasks, and so on.

This was a left-over from the first versions of the tree component.